dev/mock_source_data.py

```diff
@@ -0,0 +1,165 @@
+from os import environ, path
+
+import boto3
+from flask import Flask, request
+from flask_api import status
+
+
+FLASK_HOST = environ.get("FLASK_HOST", "0.0.0.0")
+FLASK_PORT = environ.get("FLASK_PORT", 5000)
+
+LOCALSTACK_URL = environ.get("AWS_ENDPOINT", "http://localstack:4566")
+BUCKET_NAME = environ.get("RETRIEVAL_BUCKET_NAME", "epid-sources-raw")
+S3_BUCKET_URL = f"s3://{BUCKET_NAME}"
+
+SOURCE_DETAILS = {
+	"origin": {
+		"url": ""
+	},
+	"format": "",
+	"automation": {
+		"parser": {
+			"awsLambdaArn": "",
+		},
+	},
+	"dateFilter": "",
+	"hasStableIdentifiers": False
+}
+
+SOURCE_NAMES_TO_PARSERS = {
+	"argentina": "argentina-argentina-ingestor-locale2e",
+	"brazil_acre": "brazil_acre-acre-ingestor-locale2e",
+	"brazil_amapa": "brazil_amapa-amapa-ingestor-locale2e",
+	"brazil_ceara": "brazil_ceara-ceara-ingestor-locale2e",
+	"brazil_distrito_federal": "brazil_distrito_federal-distrito_federal-ingestor-locale2e",
+	"brazil_espirito_santo": "brazil_espirito_santo-espirito_santo-ingestor-locale2e",
+	"brazil_goias": "brazil_goias-goias-ingestor-locale2e",
+	"brazil_para": "brazil_para-para-ingestor-locale2e",
+	"brazil_paraiba": "brazil_paraiba-paraiba-ingestor-locale2e",
+	"brazil_rio_de_janeiro": "brazil_rio_de_janeiro-rio_de_janeiro-ingestor-locale2e",
+	"brazil_santa_catarina": "brazil_santa_catarina-santa_catarina-ingestor-locale2e",
+	"brazil_srag": "brazil_srag-srag-ingestor-locale2e",
+	"canada": "canada-canada-ingestor-locale2e",
+	"ch_zurich": "ch_zurich-zurich-ingestor-locale2e",
+	"colombia": "colombia-colombia-ingestor-locale2e",
+	"cuba": "cuba-cuba-ingestor-locale2e",
+	"czechia": "czechia-czechia-ingestor-locale2e",
+	"england": "england-england-ingestor-locale2e",
+	"estonia": "estonia-estonia-ingestor-locale2e",
+	"example": "example-example-ingestor-locale2e",
+	"germany": "germany-germany-ingestor-locale2e",
+	"hongkong": "hongkong-hongkong-ingestor-locale2e",
+	# TODO: add India (breaking due to two sample_data files)
+	"japan": "japan-japan-ingestor-locale2e",
+	"mexico": "mexico-mexico-ingestor-locale2e",
+	"new_zealand": "new_zealand-new_zealand-ingestor-locale2e",
+	"paraguay": "paraguay-paraguay-ingestor-locale2e",
+	"peru": "peru-peru-ingestor-locale2e",
+	"republic_of_korea": "republic_of_korea-republic_of_korea-ingestor-locale2e",
+	"riograndedosul": "riograndedosul-riograndedosul-ingestor-locale2e",
+	"saopaolo": "saopaolo-saopaolo-ingestor-locale2e",
+	"scotland": "scotland-scotland-ingestor-locale2e",
+	"south_africa": "south_africa-south_africa-ingestor-locale2e",
+	"taiwan": "taiwan-taiwan-ingestor-locale2e",
+	"thailand": "thailand-thailand-ingestor-locale2e",
+	"usa": "USA-USA-ingestor-locale2e",
+	"variants": "variants-variants-ingestor-locale2e",
+}
+
+SOURCE_IDS_TO_NAMES = {}
+
+UPLOAD_RESPONSE = {
+	"_id": "".join("0" for i in range(24))
```

Comment:
“0” * 24 works as well :)

Reply:
It does seem a bit suspect in a dynamically-typed language. The [Zen of Python](https://www.python.org/dev/peps/pep-0020/#id2) says to favor explicit over implicit, so I think I slightly prefer `range`.